authors/apps/authentication/serializers.py

```diff
@@ -99,6 +98,7 @@ def validate(self, data):
                 'This user has been deactivated.'
             )
         token = generate_jwt_token(email)
+
         print(token)
```

Comment:
@mikenthiwa please remember to remove all `debug` related code from your submissions.